class.jetpack-gutenberg.php

```diff
@@ -1,13 +1,14 @@
 <?php //phpcs:ignore WordPress.Files.FileName.InvalidClassFileName
+namespace Automattic\Jetpack;
+
+use Automattic\Jetpack;
```

Comment:
Having the same full qualified class name like the namespace (`Automattic\Jetpack`) can be confusing - do you think we should define the class as `Automattic\Jetpack\Jetpack` or `Automattic\Jetpack\Plugin` or `Automattic\Jetpack\Main` instead?